featurizers/fingerprint_helper_functions.py

```diff
@@ -0,0 +1,138 @@
+"""
```

Comment:
I'm not sure if this should be a one huge file. Maybe we should break fingerprints into separate files? MAP4 and MHFP code is quite large in itself, and `fingerprints.py` file will grow quickly at the current rate.

Reply:
Changed name to `map4_mhfp_helper_functions.py`. The idea from the beginning was, that here would be only helper functions for those 2 fingerprints, but the name was misleading.